libpv/include/pv/nic.h

```diff
@@ -51,5 +64,21 @@ bool pv_nic_tx(uint16_t nic_id, uint16_t queue_id, struct pv_packet* pkt);
  */
 uint16_t pv_nic_tx_burst(uint16_t nic_id, uint16_t queue_id, struct pv_packet** pkts, uint16_t nb_pkts);
 
+/**
+ * Check tx offload is enabled on config.
+ * 
+ * @return
+ *   true if enabled
+ */
+bool inline pv_tx_offload_enabled(struct pv_nic nic, uint32_t feature);
```

Comment:
Let's use is_xxx instead of just xxx
pv_tx_offload_enabled -> pv_is_tx_offload_enabled